lib/CMakeLists.txt

```diff
@@ -318,12 +318,13 @@ message(STATUS "Available machines: ${available_machines}")
 file(GLOB xml_files ${PROJECT_SOURCE_DIR}/gen/*.xml)
 file(GLOB py_files ${PROJECT_SOURCE_DIR}/gen/*.py)
 file(GLOB h_files ${PROJECT_SOURCE_DIR}/kernels/volk/*.h)
+file(GLOB tmpl_files ${PROJECT_SOURCE_DIR}/kernels/volk/*.tmpl)
```

Comment:
I assume the changes in `lib/CMakeLists.txt` are obsolete now.

Reply:
Fixed